go/common/persistent/persistent.go

```diff
@@ -50,6 +50,8 @@ func NewCommonStore(dataDir string) (*CommonStore, error) {
 	opts = opts.WithLogger(ekbadger.NewLogAdapter(logger))
```

Comment:
Not changing the `ekbadger` import here?

Reply:
Oops, missed that somehow. Fixed now, thanks!